docs/site/Validation.md

```diff
@@ -0,0 +1,36 @@
+---
+lang: en
+title: 'Validation'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Validation.html
+---
+
+Within a LoopBack application, validation can be added in various places
+depending on the usage. Some types of validations come out-of-the-box in
+LoopBack, such as type validation in the REST layer, whereas some require
+additional configuration or code.
+
+There are various types of validations such as:
+
+- validation of input/output for method invocations
+- validation of model instance properties, for example, age < 0
+- validation of model collections, for example, uniqueness
+
+Let's take a closer look at how validation can be added in the following layers:
+
+- [REST layer](Validation-REST-layer.md)
+- [Controller layer](Validation-controller-layer.md)
```

Comment:
maybe have the description consistent with the url and title?
> Validation in the Controller, Repository and Service Layer

Reply:
What would be your suggested change? 
If I understand your comment correctly, since the sentence before the bullets are “… look at how validation can be added in the following layers:“, so it won’t make sense to make the bullet as
from REST layer to validation in REST layer.

Reply:
oh let me clarify, I meant using "Controller, Repository and Service layer"